test/test_util_astrcmp.py

```diff
@@ -20,9 +19,15 @@ def test_astrcmp(self):
         self.assertAlmostEqual(0.7083333333333333, astrcmp(u"The Great Gig in the Sky", u"Great Gig In The sky"))
 
 
-class AstrcmpCTest(AstrcmpBase, unittest.TestCase):
-    func = astrcmp_c
-
+from picard.util.astrcmp import astrcmp_py
```

Comment:
Because it goes with the test immediately below and compares and contrasts with the one for the C module.